### PR TITLE
Feature/api changes

### PR DESCRIPTION
**Pull request by vasanth9**
related to #5309

### CODE REVIEW

**Comment by AugustinMauroy** [comment]
To render correctly you have to migrate [tables.scss](https://github.com/nodejs/nodejs.dev/blob/main/src/styles/tables.scss).

**Comment by ovflowd** [comment]
Hey @vasanth9 could you recreate your branch? It seems to have unrelated changes from your previous PRs 👀

File: i18n/locales/en.json [hunk 1]
```diff
@@ -52,6 +52,10 @@
   "components.footer.links.github": "GitHub",
   "components.prevnextlink.next": "NEXT",
   "components.prevnextlink.previous": "PREV",
+  "docs.api.addedIn": "Added in: {version}",
```
Comment (ovflowd) [inline i18n/locales/en.json:55]: The path of translation keys should be: `components.api.apiChanges.{keyName}`

File: components/Api/ApiChanges/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.changes-component {
```
Comment (ovflowd) [inline components/Api/ApiChanges/index.module.scss:1]: Can we camelCase this classname?

File: components/Api/ApiChanges/index.module.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  cursor: pointer;
+
+  strong {
+    display: inline-block;
+    font-size: 0.67em;
+    margin: 0;
+    text-transform: uppercase;
+  }
+}
```
Comment (ovflowd) [inline components/Api/ApiChanges/index.module.scss:10]: Missing line ending here 🤔

File: components/Api/ApiChanges/index.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import { parseApiDocsVersion } from '../../../util/parseApiDocsVersion';
+import type { ApiChange, ApiUpdate } from '../../../types';
+import type { FC } from 'react';
+
+type ChangesProps = {
+  update: ApiUpdate;
+  changes: ApiChange[];
+};
+
+const Changes: FC<ChangesProps> = ({ update, changes }) => (
```
Comment (ovflowd) [inline components/Api/ApiChanges/index.tsx:12]: Can we name this component `ApiChanges`? to be consistent with the name of the folder 🤔

**Comment by ovflowd** [comment]
Overall it looks great. I'd add more stories with the other kinds of ApiChanges that are available.

Also, no unit tests are needed as the only one would be the snapshot test, and this will be covered by https://github.com/nodejs/nodejs.org/pull/5340

**Comment by ovflowd** [comment]
Also don't forget to rebase your branch!

**Review by ovflowd — COMMENTED** [review]
So far really good work! Thank you for the contribution! 🙇

**Comment by ovflowd** [comment]
Also, this should probably look more like this. (Like a table when expanded) And probably some different font-sizes. I'd recommend checking if something is wrong here. (Maybe a rebase would suffice, but otherwise check your stuff)

image

File: i18n/locales/en.json (hunk 1 above)
Comment (vasanth9) [reply i18n/locales/en.json:55]: ok

File: components/Api/ApiChanges/index.module.scss (hunk 2 above)
Comment (vasanth9) [reply components/Api/ApiChanges/index.module.scss:1]: ok

File: components/Api/ApiChanges/index.tsx (hunk 4 above)
Comment (vasanth9) [reply components/Api/ApiChanges/index.tsx:12]: ok

**Comment by AugustinMauroy** [comment]
@ovflowd 

> Also, this should probably look more like this. (Like a table when expanded) And probably some different font-sizes. I'd recommend checking if something is wrong here. (Maybe a rebase would suffice, but otherwise check your stuff)

solution is [here](https://github.com/nodejs/nodejs.org/pull/5344#issuecomment-1529480522)

**Comment by ovflowd** [comment]
**This PR got accidentally closed because the branch got deleted. Please feel free to reopen the PR.**

**Comment by ovflowd** [comment]
**This PR got accidentally closed because the branch got deleted. Please feel free to reopen the PR.**

**Comment by shanpriyan** [comment]
Closing this in favour of #5351

**Pull request closed without merging**